packages/flutter/lib/src/material/slider.dart

```diff
@@ -212,41 +214,55 @@ class Slider extends StatefulWidget {
 
 class _SliderState extends State<Slider> with TickerProviderStateMixin {
   static const Duration enableAnimationDuration = const Duration(milliseconds: 75);
-  static const Duration positionAnimationDuration = const Duration(milliseconds: 75);
-
-  // Animation controller that is run when interactions occur (taps, drags,
-  // etc.).
-  AnimationController reactionController;
+  static const Duration valueIndicatorAnimationDuration = const Duration(milliseconds: 100);
+
+  // Animation controller that is run when the overlay is shown in response
```

Comment:
It might be helpful to say "overlay (radial reaction)" here, since overlay is pretty generic. Unless you're really steeped in the slider spec of course.